src/routes/graph/update-pin-groups.ts

```diff
@@ -0,0 +1,39 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+
+interface Request {
+  body: {
+    hashIds: string[];
+    symbolKey?: string;
+    gridHashIds?: string[];
```

Comment:
These fields are present here, but i don't think I saw the matching implementation in the platform. And given the messages (from Rob), I think this should be removed right ?

Reply:
You're right, updated it